### PR TITLE
feat: drag and drop image in editor(markdown, wysiwyg)

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* added D&D operation for processing the image in editor

**markdown**

**wysiwyg**

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/src/helper/image.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    reader.onload = (event) => callback(event.target!.result as string);
+    reader.readAsDataURL(blob);
+  });
+}
+
+export function emitImageBlobHook(
+  eventEmitter: Emitter,
+  editorType: EditorType,
+  blob: File,
+  type: string
```
Comment (seonim-ryu) [inline apps/editor/src/helper/image.ts:17]: 타입 추가할 수 있지 않을까요?? 제가 확인했을 때는 `'ui' | 'drop'` 타입이 2가지인 것 같네요~

File: apps/editor/src/helper/image.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { EditorType, HookCallback } from '@t/editor';
+import { Emitter } from '@t/event';
+
+export function addDefaultImageBlobHook(eventEmitter: Emitter) {
+  eventEmitter.listen('addImageBlobHook', (blob: File, callback: HookCallback) => {
+    const reader = new FileReader();
+
+    reader.onload = (event) => callback(event.target!.result as string);
```
Comment (seonim-ryu) [inline apps/editor/src/helper/image.ts:8]: `target` 바로 디스트럭처링 할 수 있겠네요ㅎ
```js
({target}) => callback(target!.result as string);
```

**Review by seonim-ryu — APPROVED** [review]
[1/29] 리뷰 완료합니다. 고생하셨어요!

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

File: apps/editor/src/helper/image.ts (hunk 1 above)
Comment (js87zz) [reply apps/editor/src/helper/image.ts:17]: 넵 근데 타입을 좁히면 타입스크립트에서 이벤트 `type`이 string으로 정해져 있어서..에러가 나거든요. 그렇다고 타입 어썰션으로 강제 변환하기는 애매해서 string으로 두었습니다.

**Pull request merged**